packages/grid/_modules_/grid/hooks/utils/useOptionsProp.ts

```diff
@@ -35,10 +29,10 @@ export function useOptionsProp(apiRef: GridApiRef, props: GridComponentProps): G
       ...props,
       classes: composeClasses(
```

Comment:
If it's in the critical path, we could move the computation of `classes` to a different useMemo. We don't depend on the props, only props.classes to update it.

Reply:
Oh actually, @flaviendelangle noticed it too in #2292